src/Diagnostics.RuntimeHost/Controllers/ApiManagement/ApiManagementController.cs

```diff
@@ -35,6 +35,12 @@ public ApiManagementController(IStampService stampService, ICompilerHostClient c
             return await base.GetDetector(GetResource(subscriptionId, resourceGroupName, serviceName), detectorId, startTime, endTime, timeGrain);
         }
 
+        [HttpPost(UriElements.Detectors + UriElements.DetectorResource + UriElements.Statistics + UriElements.StatisticsResource)]
+        public async Task<IActionResult> GetSystemInvoker(string subscriptionId, string resourceGroupName, string serviceName, string invokerId, string detectorId, [FromBody] dynamic postBody, string startTime = null, string endTime = null, string timeGrain = null)
```

Comment:
The 'invokerId' and 'detectorId' are not in order of the url. I dont think it matters but just for consistency sake, have them in order